src/aroma/aroma.c

```diff
@@ -158,8 +158,11 @@ byte libaroma_start() {
   
 #ifdef LIBAROMA_TTY_KDSETMODE
   ALOGI("KDSETMODE = KD_GRAPHICS");
-  int tty1 = open("/dev/tty1", O_RDWR);
+  char ttydev[32];
+  snprintf(ttydev,32,"/dev/tty%i",LIBAROMA_TTY_KDSETMODE);
```

Comment:
while I generally prefer using snprintf, in this case you know the length so you could just use `sprintf`.
Alternatively (what I'd prefer) since the function `libaroma_start` has an error code return value already anyway you could properly check the return value just in case we overlooked something.

A proper check would be:
```c
int rc = snprintf(buf, len, ...);
if (rc < 0 || (size_t)rc >= len) {
    // error
}
```